src/backends/kubernetes/kubernetes_backend.py

```diff
@@ -487,8 +523,11 @@ def create_pod_object(
             ),
         )
 
+    def get_service_name(self, tank_index: int) -> str:
```

Comment:
Glad to see this here, I have it as an unrelated commit in a different branch. Hope to use this instead